### PR TITLE
Remove deprecated options in UI since 17.1 and earlier

### PR DESCRIPTION
**Pull request by mrjohn0011**

### CODE REVIEW

File: js/ui/tooltip/tooltip.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ var Tooltip = Popover.inherit({
             /**
             * @name dxTooltipOptions_toolbarItems
             * @publicName toolbarItems
-            * @hidden
+             * @hidden
```
Comment (San4es) [inline js/ui/tooltip/tooltip.js]: Revert this change?

Comment (mrjohn0011) [reply js/ui/tooltip/tooltip.js]: Yes, this change is redundant

**Review by nightskylark — DISMISSED** [review]

**Review by nightskylark — COMMENTED** [review]
LGTM

**Comment by jsdmitry** [comment]
LGTM

**Review by lukyanovas — DISMISSED** [review]

**Review by vconst — DISMISSED** [review]

**Review by nightskylark — DISMISSED** [review]

**Review by nightskylark — APPROVED** [review]
